sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/StateStoreConf.scala

```diff
@@ -26,12 +26,11 @@ private[streaming] class StateStoreConf(@transient private val conf: SQLConf) ex
 
   import SQLConf._
 
-  val maxDeltasForSnapshot = conf.getConf(STATE_STORE_MIN_DELTAS_FOR_SNAPSHOT)
```

Comment:
The current `max...` should be a typo given `STATE_STORE_**MIN**_DELTAS_FOR_SNAPSHOT`